frontend/src/store.js

```diff
@@ -46,7 +46,7 @@ const store = createStore({
   actions: {
     async createRoom({ state, commit }, name) {
       const isBasicAuth = state.config.AUTH_TYPE === 'BASIC_AUTH';
-      const conferenceType = state.conferenceRoomType || 'MESH';
+      const conferenceType = state.conferenceRoomType?.startsWith('SFU') ? 'SFU' : 'MESH';
```

Comment:
What is the reason for this change?

Reply:
It was directly sending the  'Conference Room Type' values in Preferences to the backend while creating the room.  After adding text 'MESH' => 'MESH (E2E encrypted conference)' and similarly for SFU, it was breaking the room creation message.

Reply:
I see. In that case we don't handle the dropdown correct. It's my fault. I should not have changed the value name, but  instead use labels in the dropdown. I can fix that tomorrow if you like.